recipes/gfortran_osx-64/run_test.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
+
+# we need to tell the linker where to look in our tests
+# this step is not needed when using conda build
+export DYLD_FALLBACK_LIBRARY_PATH=${PREFIX}/lib
```

Comment:
I don't understand why this is needed? Isn't `rpath` set in the activation scripts?

Reply:
This was in the old recipe. In my experience it has failed without. I think the reason is that the executable produced by the compiler itself is not run through conda build. This could not be true anymore.

Reply:
@mingwandroid, which package sets LDFLAGS in osx? Is it the C compiler?

Reply:
clang_osx-64 AFAIR. 

> `export DYLD_FALLBACK_LIBRARY_PATH`

.. should probably be removed. It does nothing on machines with SIP enabled. If it cannot be built with SIP enabled it's unlikely that is will build on ci systems, which I assume enabled SIP. What do the MacOS ci run, MacOS version wise?

Reply:
clang_osx-64 AFAIR. 

> `export DYLD_FALLBACK_LIBRARY_PATH`

.. should probably be removed. It does nothing on machines with SIP enabled. If it cannot be built with SIP enabled it's unlikely that is will build on ci systems, which I assume enabled SIP. What do the MacOS ci run, MacOS version wise?

Reply:
This has been building fine on Travis. Happy to remove.

Reply:
We use Xcode 9.4 and OS X 10.13

Reply:
What does `csrutil status` print? Would also be keen to find out on azurepipelines.